static/src/javascripts/projects/commercial/modules/dfp/render-advert-label.js

```diff
@@ -10,12 +10,31 @@ const shouldRenderLabel = adSlotNode =>
         adSlotNode.getElementsByClassName('ad-slot__label').length
     );
 
+const createAdCloseDiv = (): HTMLDivElement => {
+    const closeDiv: HTMLDivElement = document.createElement('div');
```

Comment:
This close `div` should really be a `button` as clicking it results in an actionable change to the page. A `button` element  gives an indication that this element is more than an element used for layout such as a `div`, this distinction is important as many accessibility tools parse the structure of a page to help guide their user through it and the elements used is important when denoting their  importance/role.

Here's some markup taken from another of our templates: 

```js
<button class="button engagement-banner__close-button js-site-message-close js-engagement-banner-close-button" data-link-name="hide release message">
  <span class="u-h">Close</span>
  ${closeCentralIcon.markup}
</button>
```
The `<span class="u-h">Close</span>` is  important as this  is picked up by screen readers, but isn't visible to users.

Reply:
We block screen readers on our ad elements, so apart from the close span 👍